workspaces/simon-game/src/app/components/Box.tsx

```diff
@@ -0,0 +1,16 @@
+import React from "react";
+
+type Props = { color?: string };
+
+export function Box(props: Props) {
+  const style = {
+    backgroundColor: props.color || "lightblue",
+    height: 100,
+    width: 100,
```

Comment:
It'd be cool if we can make this dynamic too 😃 (take these in as props)

Reply:
Do you think we will want to have boxes of different sizes?

Reply:
True, I don't think for this game they'll be different sizes, was just offering the customization.

Reply:
Decided to add this in for practice. We can leave them as the default value for this project

Reply:
Fair enough!